cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -143,16 +147,25 @@ public CentralAPIClient(String baseUrl, Proxy proxy, String accessToken) {
         this.verboseEnabled = Boolean.parseBoolean(System.getenv(SYS_PROP_CENTRAL_VERBOSE_ENABLED));
         this.proxyUsername = "";
         this.proxyPassword = "";
+        this.connectTimeout = 60;
```

Comment:
Wouldn't it be better to move this value to a constant that explains why this value is 60 like,

```suggestion
        this.connectTimeout = DEFAULT_CONNECT_TIMEOUT;
```